### ISSUE TITLE
option to ignore buffer in return

### ISSUE THREAD

**Issue by Dexus** [imagemin/imagemin#208]
i have the problem that i always run out of memory because i try to minify over 20.000 files in a folder.
And i dont need the buffer so is it possible to igrnor it?

**Comment by sindresorhus** [comment]
Yes, I think the API should be changed to only buffer when the `output` argument is not specified, or maybe even better, create separate methods for getting a buffer and outputting to files.
